www/include/reporting/dashboard/DB-Func.php

```diff
@@ -158,6 +158,12 @@ function getLogInDbForHostGroup($hostgroup_id, $start_date, $end_date, $reportTi
 
     $hosts_id = $centreon->user->access->getHostHostGroupAclConf($hostgroup_id, 'broker');
     if (count($hosts_id) == 0) {
+        $hostgroupStats["average"]["UP_MP"] = 0;
```

Comment:
I think labels could be added here : https://github.com/centreon/centreon/blob/ce48b838ee0c213a01e63c193b40d265bd9eee83/www/include/reporting/dashboard/common-Func.php#L116
just be sure there are no side effects